### PR TITLE
Style cleanup for `scripts/*.R` scripts

### PR DESCRIPTION
**Pull request by bpbond**
Ran through `formatR::tidy_dir`; added `@param` roxygen fields;
standardized indents; made sure all `if` blocks have braces; changed to
`paste0` where applicable; cracked down on over-length lines.

### CODE REVIEW

**Comment by bpbond** [comment]
Also changed from `1:length(x)` to `seq_along(x)`.

File: scripts/workflow.bm.R [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
-# which accompanies this distribution, and is available at
-# http://opensource.ncsa.illinois.edu/license.html
+# Copyright (c) 2012 University of Illinois, NCSA.  All rights reserved. This program and the accompanying materials are made
+# available under the terms of the University of Illinois/NCSA Open Source License which accompanies this distribution, and is
+# available at http://opensource.ncsa.illinois.edu/license.html
 #-------------------------------------------------------------------------------
 
-# ----------------------------------------------------------------------
-# Load required libraries
+# ---------------------------------------------------------------------- Load required libraries
```
Comment (robkooper) [inline scripts/workflow.bm.R]: can you leave this as 3 lines

File: scripts/workflow.bm.R [hunk 2]
```diff
@@ -78,29 +64,21 @@ kill.tunnel <- function() {
...
 }))
 
-#options(warning.expression=status.end("ERROR"))
-
 
-# ----------------------------------------------------------------------
-# PEcAn Workflow
-# ----------------------------------------------------------------------
-# Open and read in settings file for PEcAn run.
+# ---------------------------------------------------------------------- PEcAn Workflow
```
Comment (robkooper) [inline scripts/workflow.bm.R]: leave comment as is

File: scripts/workflow.pda.R [hunk 3]
```diff
@@ -1,16 +1,12 @@
...
-# which accompanies this distribution, and is available at
-# http://opensource.ncsa.illinois.edu/license.html
+# Copyright (c) 2012 University of Illinois, NCSA.  All rights reserved. This program and the accompanying materials are made
+# available under the terms of the University of Illinois/NCSA Open Source License which accompanies this distribution, and is
+# available at http://opensource.ncsa.illinois.edu/license.html
 #-------------------------------------------------------------------------------
 
-# ----------------------------------------------------------------------
-# Load required libraries
+# ---------------------------------------------------------------------- Load required libraries
```
Comment (robkooper) [inline scripts/workflow.pda.R]: leave comment as it was

File: scripts/workflow.pda.R [hunk 4]
```diff
@@ -19,216 +15,212 @@ library(RCurl)
...
-#options(warning.expression=status.end("ERROR"))
+# options(warning.expression=status.end('ERROR'))
 
 
-# ----------------------------------------------------------------------
-# PEcAn Workflow
-# ----------------------------------------------------------------------
-# Open and read in settings file for PEcAn run.
-if (is.na(args[1])){
+# ---------------------------------------------------------------------- PEcAn Workflow
```
Comment (robkooper) [inline scripts/workflow.pda.R]: leave comment as it was

File: scripts/workflow.pda.R [hunk 5]
```diff
@@ -19,216 +15,212 @@ library(RCurl)
...
       }
     }
   }
-  saveXML(listToXml(settings, "pecan"), file=file.path(settings$outdir, 'pecan.METProcess.xml'))
-
+  saveXML(listToXml(settings, "pecan"), file = file.path(settings$outdir, "pecan.METProcess.xml"))
+  
   # Check status to avoid repeating work
-  check.status <- function(check.name){
-    status.file=file.path(settings$outdir, "STATU")
```
Comment (robkooper) [inline scripts/workflow.pda.R]: should be STATUS

File: scripts/workflow.pda.R [hunk 6]
```diff
@@ -19,216 +15,212 @@ library(RCurl)
...
-if (check.status("SENSITIVITY") == 0){
+if (check.status("SENSITIVITY") == 0) {
   status.start("SENSITIVITY")
   run.sensitivity.analysis()
   status.end()
 }
 
 # Run parameter data assimilation
-if(('assim.batch' %in% names(settings))) {
+if (("assim.batch" %in% names(settings))) {
```
Comment (robkooper) [inline scripts/workflow.pda.R]: can remove parenthesis (no need for double)

**Comment by robkooper** [comment]
@bpbond Any specific argument you used with formatR::tidy_dir

**Comment by bpbond** [comment]
@robkooper just `arrow=TRUE`. Hmm, sorry about those comment line changes--I'll keep an eye on `formatR` in the future.

**Pull request merged**